File: joseffe-barroso/Python/aula04/for/exercise40.py
index = 1

firstNumber = -1
secondNumber = 1
thirdNumber = 1
sumNumbers = 0

for index in range(1, 31, 1):
    print(secondNumber)
    sumNumbers = secondNumber + firstNumber + thirdNumber
    firstNumber = secondNumber
    secondNumber = thirdNumber
    thirdNumber = sumNumbers
    index = index + 1

File: joseffe-barroso/Java/src/aula04/For/Exercise40.java
package For;

public class Exercise40 {

	public static void main(String[] args) {
		int index = 1, firstNumber = -1, secondNumber = 1, thirdNumber = 1, sumNumbers = 0;

		for (index = 1; index < 31; index++) {
			System.out.printf("\n%d", secondNumber);
			sumNumbers = secondNumber + firstNumber + thirdNumber;
			firstNumber = secondNumber;
			secondNumber = thirdNumber;
			thirdNumber = sumNumbers;
			index = index + 1;
		}
	}

}